packages/btcindexer/src/btcindexer.ts

```diff
@@ -375,27 +408,37 @@ export class Indexer implements Storage {
 		}
 
 		if (mintBatchArgs.length > 0) {
-			console.log(`Minting: Sending batch of ${mintBatchArgs.length} mints to SUI...`);
+			console.log({
+				msg: "Minting: Sending batch of mints to Sui",
+				count: mintBatchArgs.length,
+			});
 			const suiTxDigest = await this.nbtcClient.tryMintNbtcBatch(mintBatchArgs);
 			const now = Date.now();
 			if (suiTxDigest) {
-				console.log(`Sui transaction successful. Digest: ${suiTxDigest}`);
+				console.log({ msg: "Sui batch mint transaction successful", suiTxDigest });
 				const setMintedStmt = this.d1.prepare(
 					`UPDATE nbtc_minting SET status = 'minted', sui_tx_id = ?, updated_at = ? WHERE tx_id = ? AND vout = ?`,
 				);
 				const updates = processedPrimaryKeys.map((p) =>
 					setMintedStmt.bind(suiTxDigest, now, p.tx_id, p.vout),
 				);
-				console.log(`Minting: Updating status for ${updates.length} transactions in D1.`);
+				console.log({
+					msg: "Minting: Updating status to 'minted' in D1",
+					count: updates.length,
+				});
 				await this.d1.batch(updates);
```

Comment:
handle result, and log error if an error (add TODO & a new task)

Reply:
done